portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -42,26 +44,35 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         Query query = new Query("Comment");
         PreparedQuery results = datastore.prepare(query);
         
-        List<String> texts = new ArrayList<>();
+        List<String[]> cmts = new ArrayList<>();
```

Comment:
I'd recommend using an object instead of a string array, but it should work as is, at least for now. Consider updating this with a class. A good example showing that is the [`todo-list` example](https://github.com/hosungs/sps/tree/master/walkthroughs/week-2-server/examples/todo-list/src/main/java/com/google/sps) in week 2 walkthroughs.